src/main/app/user_registration/mutations.cljs

```diff
@@ -0,0 +1,55 @@
+(ns app.user-registration.mutations
+  (:require [com.fulcrologic.fulcro.algorithms.merge :as merge]
+            [com.fulcrologic.fulcro.mutations :refer [defmutation]]
+            [taoensso.timbre :as log]
+            [app.user-registration.model.validation :as v]
+            [app.user-registration.model :as m]))
+
+(defmutation update-input
+  [{registration-id :user-registration/id :as params}]
+  (action [{:keys [state]}]
+    (let [data-to-update (select-keys params [:user-registration/email :user-registration/password :user-registration/confirm-password])]
+      (swap! state #(update-in % [:user-registration/id registration-id] merge data-to-update)))))
+
+(defn split-affected-errors
```

Comment:
Namespaces `app.user-registration.model` seems to be a better place for this function